test/regress/regress1/quantifiers/dump-inst-proof.smt2

```diff
@@ -1,5 +1,5 @@
 ; REQUIRES: proof
-; COMMAND-LINE: --dump-instantiations --proof --print-inst-full
+; COMMAND-LINE: --dump-instantiations --produce-unsat-cores --print-inst-full
```

Comment:
I like this change.  It didn't make much sense to use `--proof` for this feature.